sorc/orog.fd/lake.fd/inland.F90

```diff
@@ -0,0 +1,246 @@
+!<><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
```

Comment:
Can this program run one tile at a time?  Or do you need to process all six tiles?

Reply:
Since all six must be processed, I may want to move the lake processing to its own scripts, instead of in fv3gfs_make_orog.sh.   That way we can continue to use CFP on the Cray.